pkg/bmc/idrac.go

```diff
@@ -46,11 +46,12 @@ func (a *iDracAccessDetails) Driver() string {
 // pre-populated with the access information, and the caller is
 // expected to add any other information that might be needed (such as
 // the kernel and ramdisk locations).
-func (a *iDracAccessDetails) DriverInfo(bmcCreds Credentials) map[string]interface{} {
+func (a *iDracAccessDetails) DriverInfo(bmcCreds Credentials, verifyCA bool) map[string]interface{} {
```

Comment:
It seems like it would be cleaner if we pass the flag to the constructor instead of this method. If we ever have the go code talking directly to the BMC, we're going to need the flag there anyway.